examples/hello/test.bats

```diff
@@ -3,6 +3,7 @@ load "${CHTEST_DIR}/common.bash"
 setup () {
     scope standard
     prerequisites_ok hello
+    ch_img=${CH_TEST_IMGDIR}/${ch_tag}
```

Comment:
Shouldn't this use `ch_imgdir`? And is it different from the default value of `ch_img`?